lib/apivore/rspec_builder.rb

```diff
@@ -32,7 +32,7 @@ def apivore_build_path(path, data)
           raise URI::InvalidURIError, "No substitution data found for {#{key}} to test the path #{path}.\nAdd it via an:\n  apivore_setup '<path>', '<method>', '<response>' do\n    { '#{key}' => <value> }\n  end\nblock in your specs.", caller
         end
       end
-      path
+      path + (data['_query_string'] ? "?#{data['_query_string']}" : '')
```

Comment:
@gwshaw, I've figured it out, the setups sometimes don't return a sensible hash at all, see https://github.com/westfieldlabs/apivore/blob/master/spec/request_spec.rb#L26
I've just changed the code to handle this better, I just need to test it properly and refactor some more before raising another PR. I'll keep you posted.